dpnp/dpnp_iface_arraycreation.py

```diff
@@ -97,6 +97,78 @@
 ]
 
 
+def _get_empty_array(
+    a,
+    /,
+    *,
+    dtype=None,
+    order="K",
+    shape=None,
+    device=None,
+    usm_type=None,
+    sycl_queue=None,
+):
+    """
+    Get an empty array as the base for empty_like, ones_like, zeros_like,
+    and full_like.
+
+    """
+    strides = None
+    if shape is None:
+        _shape = a.shape
+    elif dpnp.isscalar(shape):
+        _shape = (shape,)
+    else:
+        _shape = shape
+    _dtype = a.dtype if dtype is None else dtype
+    _usm_type = a.usm_type if usm_type is None else usm_type
+    _sycl_queue = dpnp.get_normalized_queue_device(
+        a, sycl_queue=sycl_queue, device=device
+    )
+
+    if order is None:
+        order = "C"
```

Comment:
`None` value should mean `K`
```suggestion
        order = "K"
```